src/args/project-path/default.nix

```diff
@@ -7,6 +7,6 @@ if hasPrefix "/" rel
 then
   (builtins.path {
     name = if rel == "/" then "src" else builtins.baseNameOf rel;
-    path = projectSrc + rel;
+    path = ((builtins.unsafeDiscardStringContext projectSrc) + rel);
```

Comment:
We also applied this same patch before: 
https://gitlab.com/fluidattacks/product/-/blob/a5c3d17a5e0743bfbd243b9e7aa28ea8e1ae2440/flake.nix#L55
So it should work well at scale